opensfm/features.py

```diff
@@ -313,3 +315,64 @@ def build_flann_index(features, config):
                         iterations=config['flann_iterations'])
 
     return context.flann_Index(features, flann_params)
+
+
+FEATURES_VERSION = 1
+FEATURES_HEADER = 'OPENSFM_FEATURES_VERSION'
+
+
+def load_features(filepath, config):
+    """ Load features from filename """
+    s = np.load(filepath)
+    version = _features_file_version(s)
```

Comment:
Should we get the version from s['OPENSFM_FEATURES_VERSION'] instead?

Reply:
It's what `_features_file_version` does. I do agree that I use a var named FEATURES_HEADER that contains the string `OPENSFM_FEATURES_VERSION`, which is a bit conflated. Let me know if you prefer direct use of `OPENSFM_FEATURES_VERSION` instead.

Reply:
ok, i see. I got confused by the name HEADER and though the function was looking for a file header.
looks good